test-data/unit/README.md

```diff
@@ -29,7 +29,11 @@ Add the test in this format anywhere in the file:
 - `# E: abc...` indicates that this line should result in type check error
 with text "abc..."
 - note a space after `E:` and `flags:`
-- lines without `# E: ` should cause no type check errors
+- `# E:12` adds column number to the expected error
+- `# E:-1` means that the error was expected 2 lines ago (helpful for multiple errors in a line)
```

Comment:
@JelleZijlstra  Yup ;) 
@ilevkivskyi  One error is about `TypedDict` not matching the `dict` structure. But it then infers the wrong `TypedDict` anyway (to keep going), and that doesn't fit the function call. I can probably change it to make it one error, but it would require more logic elsewhere in the code.